scripts/faucet-deploy

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# shellcheck source=lib/common.sh
+. "${LIB_DIR:-$(cd "${0%/*}/../lib"&&pwd)}/common.sh"
+
+# Deploy Token Faucet (no solc optimization)
+dappBuild token-faucet
+if [[ -z "$FAUCET" ]]; then
+    FAUCET=$(dappCreate token-faucet TokenFaucet "$(seth --to-uint256 "$(seth --to-wei 50 ETH)")")
```

Comment:
Should the amount be configurable?

Reply:
I would think so, but it was not necessary for this PR and would be more of a feature for the deployment scripts, so I think we should leave it out of this PR.  I'm happy to open a new one for this change, but I'm not sure its a feature we need/want now.  Do you know?